napari/layers/labels/labels.py

```diff
@@ -262,6 +266,19 @@ def n_dimensional(self, n_dimensional):
         self._n_dimensional = n_dimensional
         self.events.n_dimensional()
 
+    @property
+    def contour(self):
+        """bool: displays contours of labels instead of shaded regions."""
+        return self._contour
+
+    @contour.setter
+    def contour(self, contour, refresh=True):
```

Comment:
Can we drop the `refresh=True` from the setter and always refresh. I think we do that in most of our other setters and I don't see a need to special case here

Reply:
Done!